### ISSUE TITLE
Add rss? 

### ISSUE THREAD

**Issue by dalanmiller** [rocx/rocx.github.io#1]
Hey there! Just saw your first blog post via Lobste.rs[0] and definitely experiment this same issue, although I solve it by partitioning via work laptop vs personal laptop. 

I wanted to add your blog to my feeds and noticed you didn't have RSS set up yet, you should consider adding it![1]

[0] - https://lobste.rs/s/dtrkna/leading_double_life_on_github
[1] - https://joelglovier.com/writing/rss-for-jekyll

**Comment by rocx** [comment]
A syndication feed's been on my to-do list almost since its inception. The trick's getting around to it and remembering to watch your own repos for when people submit issues or pull requests. Strangely not a default thing... Glad you're interested anyway. Unfortunately, I had to take down the article mentioned (due to it being in *technical* violation of GitHub [terms of service][tos]) and there are no other articles to boast, hence the empty feed. Atom feed's available at https://rocx.rocks/feed.atom.xml.

[tos]: https://help.github.com/en/github/site-policy/github-terms-of-service#3-account-requirements

Also today, I learned GitHub automatically closes issues if they're mentioned in commit messages. Rude presumption on its part.
